WebApp/autoreduce_webapp/static/css/base.css

```
@@ -280,8 +280,4 @@ search#run_search {
 
```

Could you add something like what i have suggested below to an appropriate location within base.css to stop the automatic wrapping of `pre` blocks as code displayed in help pages is wrapping and hard to read. 

```suggestion
pre code {
  white-space: pre;
  overflow-x: auto;
  display: inline-block;
  min-width: 100%;
}
```